### ISSUE TITLE
FTB Genesis Liquid Pink Slime Recipe

### ISSUE THREAD

**Issue by nitrojector** [FTBTeam/FTB-Modpack-Issues#4227]
### Modpack

FTB Genesis

### Modpack version

1.1.1

### Log Files

None

### Describe the bug

Liquid pink slime cannot be melted from pink slime balls in *Cyclic* melting chamber -- produces regular liquid slime instead.

![image](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/23587896/47765bd4-a274-4c9b-9278-c6e4a02b3638)

### Steps to reproduce

1. Launch the game
2. Attempt to melt liquid pink slime

### Expected behaviour

Melting pink slime balls should produce liquid pink slime

**Comment by Ratlanta** [comment]
You can create a slime bee and then feed it pink slime (made in pressure chamber) to get pink slime bee. The combs produce liquid pink slime.

**Comment by nitrojector** [comment]
> You can create a slime bee and then feed it pink slime (made in pressure chamber) to get pink slime bee. The combs produce liquid pink slime. 

I don't believe there is a recipe for that either. There is one from the slime bee, but the slime bee itself has no recipe.

**Comment by Ratlanta** [comment]
![image](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/59396659/00a66055-a4ce-4300-8003-77b896c59e77)
